README.rst

```diff
@@ -90,10 +90,11 @@ Here's a full example with the libvirt provider:
          # use single quotes to avoid YAML parsing as dict due to ':'
          - 'vm.synced_folder ".", "/vagrant", type: "rsync"'
        # Dictionary of `config` options. Note that string values need to be
-       # explicitly enclosed in quotes.
+       # explicitly enclosed in quotes. To protect the quotes from being parsed
+       # away we need to quote string values twice.
        config_options:
          ssh.keep_alive: yes
-         ssh.remote_user: 'vagrant'
+         ssh.remote_user: "'vagrant'"
```

Comment:
I do not find this approach acceptable. It is a username and nobody should be forced to add the quotes. Instead of doing this we need to fix the code to add the quotes itself.

Reply:
If we are about to change something in the doc is to remove the single quotes, in that particular case no quoting is the same as double or single ones as YAML loader will load just `vagrant` text for each.

Reply:
yeah, the doc has to be updated. I've not done it in the ``vagrant.yml`` removal one, since things should still be working with them. The aim was to wait for few releases so that people migrate to proper configuration and then kill the compatibiliy. Nevertheless, I should have updated the doc about the migration path. Not sure what's the best thing to do now...